src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/unnecessaryparentheses/InputUnnecessaryParenthesesCheckSwitchExpression.java

```diff
@@ -46,16 +46,16 @@ MathOp2 tooManyParens2(int k) {
         MathOp2 e = (a, b) -> (a + b); // violation 'parentheses around assignment right-hand side'
       }
       case (2) -> { // violation 'Unnecessary parentheses around literal '2''
-        MathOp2 f = (int a, int b) -> (a + b); // violation 'paren.* around assignment right.*side'
+        MathOp2 f = (int a, int b) -> (a + b); // violation 'paren.* around assignment right.* side'
```

Comment:
Please revert change here, or make a separate `minor` commit for formatting of violation message; this is not related to check update.

Reply:
fixed

Reply:
Resolved.